leshan-server-cluster/src/main/resources/log4j2.xml

```diff
@@ -32,6 +32,9 @@ Contributors:
     <logger name="org.eclipse.leshan.server.security.SecurityCheck" level="trace" additivity="false">
         <AppenderRef ref="STDOUT"/>
     </logger>
+    <logger name="org.eclipse.leshan.core.model.LwM2mModel" level="trace" additivity="false">
```

Comment:
There is no trace logs in the LwM2mModel class (almost no logs at all).
You mean ObjectLoader?